doc/source/quickstart.rst

```diff
@@ -0,0 +1,422 @@
+KIWI Quickstart
+===============
+
```

Comment:
Although I reviewed it, I missed that part obviously. I corrected the 2nd Quickstart into "Example Appliance Descriptions".

Well, basically, my idea was to convert the content inside the `README.md` into RST and include it into the other documentation.